fab/utils.py

```diff
@@ -179,7 +179,7 @@ def traceback_string():
         trace=trace,
         type=exc_type.__name__,
         exc=exc,
-    )
+    ).replace('"', '\\"')
```

Comment:
It's not clear what you're escaping for here, so maybe this isn't the correct place to be doing it? It seems like you'd want to do this at the place where you're invoking the command. Also, would it make sense to use a function like [`shlex.quote()`](https://docs.python.org/3/library/shlex.html#shlex.quote), maybe [here](https://github.com/dimagi/commcare-hq-deploy/blob/96174966ec27b17f95e7f51e353d77abda9d8f11/fab/fabfile.py#L326-L328) on `subject`, `message`, and maybe `env.environment`?